docs/supporters/sustainable.rst

```diff
@@ -1,12 +1,12 @@
-=======================
+#######################
 Sustainable Open Source
-=======================
+#######################
```

Comment:
This is something I don't yet understand with reST, what's the difference here?

Reply:
See the [Sphinx guide for Sphinx-based documentations](https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings).

'###' is meant for doc titles, '===' denotes a H3 subheading. I didn't see a formatting change though, so I think Sphinx might be promoting the highest level subheading to title and scaling every subheading accordingly...